src/app/components/button/button.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button } from 'nhsuk-react-components';
+
+export function NHSNotifyButton({
```

Comment:
what is the difference between this and the standard nhsuk-react-components Button?

Reply:
We are adding additional props to the component